staging/src/github.com/kubeedge/beehive/pkg/core/core.go

```diff
@@ -13,7 +13,7 @@ import (
 
 // StartModules starts modules that are registered
 func StartModules() {
-	beehiveContext.InitContext([]string{common.MsgCtxTypeChannel})
+	beehiveContext.InitContext([]string{common.MsgCtxTypeChannel, common.MsgCtxTypeUS})
```

Comment:
```suggestion
	beehiveContext.InitContext([]string{common.MsgCtxTypeChannel})
```

We haven't used the `MsgCtxTypeUS` context, no need to init here. If users use it, just add by themselves. Can just add some comments above.